soc/xtensa/intel_adsp/tools/cavstwist.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+# Twister integration tool for a remote ssh-accessible cAVS audio DSP
+# host (e.g. an Up Squared board running Linux). Can be used as the
+# hook for both --device-serial-pty and --west-flash, for example:
+#
+#  export CAVS_HOST=tgl2
+#  export CAVS_KEY=$HOME/otc_private_key_3k.pem
+#  export CAVS_RIMAGE=$HOME/rimage
```

Comment:
I'd add a note/comment here that this is the path to a directory containing rimage and the config directory, this bit me and I had to look harder at what was going on.